examples/iot/v1beta1/authorizer.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: iot.aws.upbound.io/v1beta1
+kind: Authorizer
+metadata:
+  annotations:
+    meta.upbound.io/example-id: iot/v1beta1/authorizer
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    authorizerFunctionArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: lambda-authorizer
+    region: us-west-1
+    signingDisabled: false
+    status: ACTIVE
+---
```

Comment:
If you can, it would be great to have an example that documents the `tokenSigningPublicKeysSecretRef`, both to validate that functionality and to provide better documentation to users.  You can just add a yaml manifest for a `v1.Secret` to this file.

Reply:
I added it using the certificate from the AWS documentation:
https://docs.aws.amazon.com/iot/latest/developerguide/custom-auth-tutorial.html#custom-auth-tutorial-authorizer-resource